tests/Core/Tokenizers/Tokenizer/RecurseScopeMapSwitchTokenScopeTest.inc

```diff
@@ -1,26 +1,26 @@
 <?php
 
 /* testSwitchNormalSyntax */
-switch ($value) {
+switch ($value) /* testSwitchNormalSyntaxScopeOpener */ {
```

Comment:
This marker is unnecessary as there is no other open curly between the `/* testSwitchNormalSyntax */` marker and the next marker which could cause the test to get confused over it.

Reply:
I removed this marker.